internal/services/contracts_events_consumer.go

```diff
@@ -521,3 +527,61 @@ func (c *ContractsEventsConsumer) handleSyntheticDeviceNodeBurnedEvent(ctx conte
 	_, err := sd.Delete(ctx, c.dbs.DBS().Writer)
 	return err
 }
+
+func (c *ContractsEventsConsumer) handleTokensTransferredForDevice(ctx context.Context, e *ContractEventData) error {
+	var args TokensTransferredForDeviceData
+	if err := json.Unmarshal(e.Arguments, &args); err != nil {
+		return err
+	}
+
+	reward := models.Reward{
+		IssuanceWeek:      1,
```

Comment:
Just assume the event has this. We should either do that issue first or invent a name now. We can always change it later.

Reply:
Or you could use https://github.com/DIMO-Network/rewards-api/blob/046c4a852e89ce4d8dc1bf62a544a40cea1a45e8/internal/services/rewards.go#L95

But I would really like to put it in the event.